### PR TITLE
Anomaly Detection with Azure Databricks on Network log Synthetic Data.

### PR DESCRIPTION
**Pull request by mabalija**
Added labs for Anomaly Detection with Azure Databricks.

### CODE REVIEW

File: data-ai/anomaly-detection.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
+---
+title: Anomaly Detection with Azure Databricks
+category: Machine Learning, Advanced Analytics, AI, Data
+date: 2018-12-30
+tags: [Anomaly Detection, Predictive Modeling, Machine Learning, Advanced Analytics, AI, Data, Databricks, Azure, Spark, ML Pipelines, Random Forests]
```
Comment (benc-uk) [inline data-ai/anomaly-detection.md]: Tags should be be all lower case, no spaces (hypens only) and limited of a maximum of FOUR per post. Do not use tags that just duplicate the category such as "ai" or "data", people will mostly be looking for azure service names so try to use those where possible, check existing tags to make sure you're not unnecessarily introducing new tags, or tags for the same thing but with slightly different name (e.g. "azure-logicapp" vs "logicapp")

File: data-ai/anomaly-detection.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
+---
+title: Anomaly Detection with Azure Databricks
+category: Machine Learning, Advanced Analytics, AI, Data
```
Comment (benc-uk) [inline data-ai/anomaly-detection.md]: Category must be same as the folder it is placed in, so in this case "data-ai" (without the quotes), all lower case. You can not place content into multiple categories, that is what tags are for

**Review by benc-uk — CHANGES_REQUESTED** [review]
I've put some comments into the commit where things need to be changed (tags and category)

**Comment by mabalija** [comment]
I have updated both Tags and Category. Please let me know if further changes are required?

**Pull request merged**